installer/install.iss

```diff
@@ -1494,7 +1496,13 @@ begin
     InputText:='Please modify this text, e.g. delete it, then save it and exit the editor.'
     SaveStringToFile(TmpFile,InputText,False);
 
-    if not ShellExec('',CustomEditorPath,CustomEditorOptions+' "'+TmpFile+'"','',SW_HIDE,ewWaitUntilTerminated,Res) then begin
+    CustomEditorExt:=ExtractFileExt(CustomEditorPath);
+    if (CompareText(CustomEditorExt,'.bat')=0) or (CompareText(CustomEditorExt,'.cmd')=0) then
```

Comment:
Do we need to do this for `.ps1` scripts, too?

Reply:
No. Powershell launches them fine even with `SW_HIDE`. They aren't on `PATHEXT` by default, though and their default associaton is to open them with `notepad`.

Reply:
Thank you for clarifying!